Packs/CommonTypes/IndicatorFields/indicatorfield-Certificate_Names.json

```diff
@@ -1,7 +1,6 @@
 {
-    "associatedToAll": false,
+    "associatedToAll": true,
```

Comment:
Are these changes you want? 
You said you only moved to a different folder.

Reply:
We decided to not change it at the end, must have missed a few, fixing.